src/modules/editData.js

```diff
@@ -0,0 +1,47 @@
+import saveToLocalStorage from './saveToDb.js';
+
+const refreshIcon = document.querySelector('.refresh-icon');
+const editData = (dataArray) => {
+  refreshIcon.addEventListener('click', () => {
+    window.location.reload();
+  });
+
+  const todoLists = document.querySelectorAll('.list');
+  const todoContent = document.querySelectorAll('.todo-content');
+  const dragIcon = document.querySelectorAll('.drag-icon');
+  const trashIcon = document.querySelectorAll('.trash-icon');
+
+  todoContent.forEach((input, item) => input.addEventListener('click', (e) => {
+    e.stopImmediatePropagation();
+    dataArray[item].description = todoContent[item].value;
+    saveToLocalStorage(dataArray);
+    todoLists[item].classList.add('active');
+    trashIcon[item].style.display = 'flex';
+    dragIcon[item].style.display = 'none';
+    todoContent[item].focus();
+    todoContent[item].style.cursor = 'text';
+    for (let i = 0; i < todoLists.length; i += 1) {
+      if (item !== i) {
+        todoLists[i].classList.remove('active');
+        trashIcon[i].style.display = 'none';
+        dragIcon[i].style.display = 'block';
+      }
+    }
```

Comment:
Kindly remove this old for loop, in this case, you can achieve this by using forEach and filter function